getUpsData.py

```diff
@@ -11,15 +11,16 @@
 
 for line in p.stdout.readlines(): #read and store result in log file
     line = line.decode("utf-8").rstrip()
-    key = line[:line.find(":")]
-    value = line[line.find(":")+2:]
+    key = line.split(':')[0]
+    value = line.split(':')[1]
 
-    if key in string_measurements:
-        value = '"' + value + '"'
-    measurement = key + "=" + value
-    if output != "":
-        measurement = "," + measurement
-    output += measurement
+    if key in measurements:
+	try: float(value) #check if the value is a number
```

Comment:
To check if `value` is a number, it's probably better to do something like:

```python
if isinstance(value, int) or isinstance(value, float):
```

Reply:
Yeah i remembered you can do `type(value)` to check type right after i posted this.